rpcs3/Emu/SysCalls/Modules/cellSysutil.cpp

```diff
@@ -677,8 +690,8 @@ int cellSysCacheMount(vm::ptr<CellSysCacheParam> param)
 
 	//TODO: implement
 	char id[CELL_SYSCACHE_ID_SIZE];
-	strncpy(id, param->cacheId, CELL_SYSCACHE_ID_SIZE);
-	strncpy(param->getCachePath, ("/dev_hdd1/cache/" + std::string(id) + "/").c_str(), CELL_SYSCACHE_PATH_MAX);
+	strncpy_s(id, param->cacheId, CELL_SYSCACHE_ID_SIZE);
```

Comment:
If Emu.GetVFS().CreateDir takes a std::string then why must it be put into a fixed size buffer of CELL_SYSCACHE_ID_SIZE? Maybe just pass std::string(param->cacheId) + "/dev_hdd1/cache/" + std::string(id) + "/" directly to Emu.GetVFS().CreateDir?